### ISSUE TITLE
Cannot reproduce results with FAKE

### ISSUE THREAD

**Issue by AntoineGourru** [jind11/TextFooler#60]
Hi Di Jin and collaborators,

First of all, thank you for this great repo, this is highly valuable to have reproducible code/data/models of this quality.

I am currently trying to reproduce the scores from your paper, everything works well, I manage to observe the same accuracy than what's pointed in your article for imdb and yelp.

Nevertheless, the FAKE bert pretrained model that I took from your drive produces random results (I get 50% accuracy while it should be 96.7)

Are you sure that you uploaded the good model on drive ?

Best regards,

Antoine

**Comment by jind11** [comment]
I think the pre-trained model should be the correct one. Could you double check whether the model parameters are initialized correctly?

**Comment by MT010104** [comment]
I met the same problem. I got 19% acc with AG bert pretrained model. Have you managed to resolve it?
